Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -77,16 +77,24 @@ def get_commit_info(cls, args, repository, **kwargs):
         commit_objects = []
         commits_to_revert = []
         commit_bugs = set()
+        commit_radars = set()
+
         for c in args.commit:
             try:
                 commit = repository.find(c, include_log=True)
             except (local.Scm.Exception, ValueError) as exception:
                 sys.stderr.write('Could not find "{}"'.format(c) + '\n')
                 return None, None, None
+
             commit_objects.append(commit)
             commits_to_revert.append(commit.hash)
-            commit_bugs.update({Tracker.from_string(i.link) for i in commit.issues if isinstance(Tracker.from_string(i.link).tracker, bugzilla.Tracker)})
-        return commit_objects, commits_to_revert, commit_bugs
+            for i in commit.issues:
+                if isinstance(i.tracker, bugzilla.Tracker):
+                    commit_bugs.add(i)
+                elif isinstance(i.tracker, radar.Tracker):
+                    commit_radars.add(i)
+
+        return commit_objects, commits_to_revert, commit_bugs, commit_radars
```

Comment:
I do wonder about this API, though I realise it's essentially just extending what's already here.

`commit_objects` already actually contains everything, so everything else in the tuple is effectively duplicating it.

Especially with a single caller, it feels like it would be just as easy to have the caller just do things like `commits_to_revert = [c.hash for c in commit_objects]`?

Reply:
That's a good point about the hashes! The bugs and radars will be used multiple times so I'd like to keep those.